recipes/xorg/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
```

Comment:
Does it work with the `cmake_find_package`... As it has no proper `libs` I would like to check the targets are created.

Reply:
> What are the outputs of this recipe when the libraries are not installed in the system? What is the feedback to the user?
```
The following packages need to be installed:
xorg-dev
Aborted due to CONAN_SYSREQUIRES_MODE=verify
Some system packages need to be installed
```